src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+import * as React from 'react';
+
+import * as Data from '../data';
+
+const fetchSettings: RequestInit = {
+    credentials: "same-origin",
+    headers: {
+        ["X-Requested-With"]: "XMLHttpRequest"
```

Comment:
I"m usually pretty 'inconsistent' with my quotes in JavaScript. It's kinda a pain that it"ll allow both kinds of quotes ;P.

Trolling aside, I'll do a quick pass to make them consistent. If I cared more, I'd set up a linter, but I think that can wait for now.